src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int START_CAPACITY = 16;
+    private static final int DEFAULT_ARRAY_EXPANSION = 2;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Node<K, V>[] heads;
+    private int capacity;
```

Comment:
let's remove this field
we don't actually need it
because we can call `buckets.length` in code where we need capacity